README.md

```diff
@@ -0,0 +1,82 @@
+# Greeter App
+This repo provides a base agent, the Greeter app and a Makefile for setting up a python based [NDK](https://learn.srlinux.dev/ndk/intro/) development environment. Read more about the approach at [learn.srlinux.dev](https://learn.srlinux.dev/ndk/guide/env/python/).
+
+## Features
+The Greeter app demonstrates how agent can interact with the whole state and config tree of SR Linux using local gNMI access and the NDK.
+- It subscribes to configuration using the NDK.
+- It retrieves the "name" from the configuration notification and update the uptime status with a hardcoded value of 8:00 using a telemetry client.
+- It gets the uptime using gNMI. The gNMI client is locally connecting using Unix Domain Socket.
+- It logs "Hello myname, my uptime is 8:00"​. 
```

Comment:
Can we make sure, that the following use case is implemented:


> the greeter service could not only return "Hello $usermame", but also fire up a gnmi get/sub for uptime leaf and return a string "Hello $username, my uptime is $uptime", where $uptime  is retrieved via gnmi client created by the agent

Reply:
Yes it does all that.

Reply:
but the doc says agent pushes a hardcoded value of 08:00, but lets do it proper with reading the value from the actual uptime

Reply:
Oh ok, you want it configurable like the name.

Reply:
not configurable, but retrieved from a real uptime leaf.
So the greeter agent with greet with a real uptime of a node

Reply:
@hellt I uploaded the changes we discussed yesterday.